### PR TITLE
Modify createDrop function to _safemint to sender

### PR DESCRIPTION
**Pull request by rayanoncyber**
Also deployed latest deployed contracts on Rinkeby and Eth

Will test then deploy latest contracts to ETH

### CODE REVIEW

File: packages/contracts/contracts/CyberDropBase.sol [hunk 1]
```diff
@@ -105,7 +105,15 @@ contract CyberDropBase is CyberTokenBase {
     LibDropStorage.layout().drops[tokenId].shareCyber = _shareCyber;
     LibDropStorage.layout().drops[tokenId].creator = payable(sender);
 
+    // Effects
```
Comment (julesGoullee) [inline packages/contracts/contracts/CyberDropBase.sol:108]: comment is duplicate, check above

File: packages/contracts/contracts/CyberDropBase.sol [hunk 2]
```diff
@@ -105,7 +105,15 @@ contract CyberDropBase is CyberTokenBase {
...
+    LibDropStorage.Drop storage drop = LibDropStorage.layout().drops[tokenId];
+
+    drop.minted.increment();
```
Comment (julesGoullee) [inline packages/contracts/contracts/CyberDropBase.sol:111]: Use `LibDropStorage.layout().drops[tokenId].minted.increment()` in place of storing local var, as it's done above, then you can remove l109

File: packages/contracts/contracts/CyberDropBase.sol [hunk 3]
```diff
@@ -133,7 +141,9 @@ contract CyberDropBase is CyberTokenBase {
       drop.priceEnd,
       drop.stepDuration
     );
+
```
Comment (julesGoullee) [inline packages/contracts/contracts/CyberDropBase.sol:144]: space space?

**Review by julesGoullee — CHANGES_REQUESTED** [review]

**Review by rayanoncyber — COMMENTED** [review]
Done

File: packages/contracts/.gitignore [hunk 4]
```diff
@@ -1,5 +1,7 @@
 node_modules
 .env
+temp
```
Comment (julesGoullee) [inline packages/contracts/.gitignore:3]: Temp, from where is it coming?

File: packages/contracts/.gitignore [hunk 5]
```diff
@@ -1,5 +1,7 @@
...
+deployments/localhost
```
Comment (julesGoullee) [inline packages/contracts/.gitignore:4]: Agree with it. Please also exclude `deployments/hardhat` and remove them

File: packages/contracts/deployments/ethereum/.chainId [hunk 6]
```diff
@@ -0,0 +1 @@
+1
```
Comment (julesGoullee) [inline packages/contracts/deployments/ethereum/.chainId:1]: Ok, on the dev branch it's called `deployements/mainnet`, better to rename to `ethereum` as it has been renamed into the hardhat.config. Make sure it's the correct deployment coming from the production branch, mainnet directory

Comment (rayanoncyber) [reply packages/contracts/deployments/ethereum/.chainId:1]: mainnet is actually polygon :/

File: packages/contracts/.gitignore (hunk 4 above)
Comment (rayanoncyber) [reply packages/contracts/.gitignore:3]: It's for me to do deployments scripts! Take out 001 and put in tmp before I want to upgrade one contract

**Review by julesGoullee — APPROVED** [review]

**Pull request merged**